lib/string-encoding.js

```diff
@@ -12,24 +8,14 @@ module.exports = {
   // mis-typed characters)
   base32: {
     /** @type {(buf: Buffer) => string} */
-    encode(buf) {
+    encode (buf) {
       return base32.encode(buf, { type: 'crockford' })
     },
     /** @type {(str: string) => Buffer} */
-    decode(str) {
+    decode (str) {
       assert(/^[a-z0-9]+$/i.test(str), 'Invalid base32 string')
       const decoded = base32.decode(str, { type: 'crockford' })
       return decoded
-    },
-  },
-  base62: {
-    /** @type {(buf: Buffer) => string} */
-    encode(buf) {
-      return base62.encode(buf)
-    },
-    /** @type {(str: string) => Buffer} */
-    decode(str) {
-      return base62.decode(str)
-    },
-  },
+    }
+  }
```

Comment:
seems like these formatting changes are not aligned with prettier - is your editor ignoring the project's configuration by any chance?